wiki/People/Beatmap_Nominators/en.md

```diff
@@ -48,6 +49,13 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
   + osu!: 1100
 + It is usually a good idea to keep their score well above these numbers as they can vary slightly depending on the amount of applicants.
 
+### How the new potential Beatmap Nominators are chosen
++ A doublecheck of the `Mod Score` and the `Kudosu Score` will be done to all applications, in order to find out if they reached the `Required Minimum Score`.
++ Modders that acquired enough score will be evaluated by the correspondant QAT members of the game mode they applied for, taking into account the following criteria:
+  + General attitude: Tone of delivery of the mods and other behavioural expectations from the [Code of Conduct](/wiki/Ranking_Criteria/Code_of_Conduct), as well as general behaviour in chat/forums.
+  + Modding abilities: Knowledge of the general [Ranking Criteria](/wiki/Ranking_Criteria) and specific criteria of each game mode. Additional abilities like Metadata, Timing, and some others will also be taken into consideration.
++ After balancing the pros and cons of each modder, a deliberation will be made whether the modder will become a Beatmap Nominator or not.
```

Comment:
"a deliberation on whether the modder becomes a Beatmap Nominator will be made"
imo discussion would be easier to understand for non-English speakers